### PR TITLE
Additional link (Boldare)

### PR DESCRIPTION
**Pull request by krzysiekhum**
Added official Boldare's communication about COVID-19 virus.

Adds the following events or companies:
 - Boldare

### Checklist

 - [ ] I have updated the event or company counts
 - [x] I have put the most recent relevant date in the "Last Update" column
 - [x] I have linked to the article about the change, not the company's website

### CODE REVIEW

File: _data/companies.yml [hunk 1]
```diff
@@ -1321,13 +1321,6 @@
   events: Restricted
   last_update: 2020-03-11
 
-- name: Tripadvisor
-  wfh: Encouraged
-  travel: Restricted
-  visitors: Restricted
-  events: Restricted
-  last_update: 2020-03-12
-
```
Comment (drewbrew) [inline _data/companies.yml]: Looks like your git client is trying to remove tripadvisor. Would you mind fixing that?

File: _data/companies.yml [hunk 2]
```diff
@@ -452,7 +452,7 @@
   visitors: Restricted
   events: Restricted
   last_update: 2020-03-10
-
+
```
Comment (drewbrew) [inline _data/companies.yml]:
```suggestion

```

Comment (drewbrew) [inline _data/companies.yml]: Also please remove this additional whitespace.

**Comment by phildini** [comment]
Closing due to rebase and staleness. Feel free to re-open after fixes, thanks for your patience.

**Pull request closed without merging**